Server/BackEndMonitoringServer/include/CService.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "CEvent.h"
+#include "Sockets/BackEndMonitoringSockets/include/CAcceptorWrapper.h"
+
+struct ServiceParameters
+{
+    const CString& name         = "BackendMonitoringService";
```

Comment:
we don't need such formatted lines according to code style

Reply:
Fixed, thnx.